sdk/tests/deqp/modules/shared/glsTextureTestUtil.js

```diff
@@ -1922,8 +1922,10 @@ glsTextureTestUtil.computeTextureLookupDiffCube = function(
             qpWatchDog_touch(watchDog);*/
 
         for (var px = 0; px < result.getWidth(); px++) {
-            var resPix = result.getPixel(px, py);
-            var refPix = reference.getPixel(px, py);
+            /** @type {Array<number>} */
+            var resPix = deMath.divide(deMath.subtract(result.getPixel(px, py), sampleParams.colorBias), sampleParams.colorScale);
+            /** @type {Array<number>} */
+            var refPix = deMath.divide(deMath.subtract(reference.getPixel(px, py), sampleParams.colorBias), sampleParams.colorScale);
 
```

Comment:
Thanks for tracking this down.

Can we do inline computation instead of using deMath?  deMath operations are the bottleneck of perf because they create new vectors at each operation.